assets/src/domain/eventEditor/ui/datetimes/dateForm/useDateFormConfig.ts

```diff
@@ -0,0 +1,126 @@
+import { __ } from '@wordpress/i18n';
+import { format, parseISO, parse } from 'date-fns';
+
+import { EspressoFormProps, dateFormat, timeFormat } from '@application/ui/forms/espressoForm';
+import useDatetimeItem from '../../../services/apollo/queries/datetimes/useDatetimeItem';
+import { EntityId } from '@edtrServices/apollo/types';
+import { PLUS_ONE_MONTH, PLUS_TWO_MONTHS } from '../../../../shared/constants/defaultDates';
+
+interface DateFormShape {
+	name?: string;
+	description?: string;
+	capacity?: number;
+	dateTime?: {
+		startDate: string;
+		startTime: string;
+		endDate: string;
+		endTime: string;
+	};
+	isTrashed?: boolean;
```

Comment:
should this be derived from Date type?

Reply:
You mean a custom type?

Reply:
I meant `Datetime` type, sorry for the confusion